aeronautical/asterix/asterix_file.ksy

```diff
@@ -0,0 +1,46 @@
+meta:
+  id: asterix_file
+  file-extension: ast
+  endian: be
+  license: GPL-3.0-only
+  imports:
+    - asterix_message
+  ks-version: 0.9
+
+doc: |
+  Implementantion of ASTERIX format.
+
+  From Eurocontrol Asterix web page at
+  https://www.eurocontrol.int/asterix
+
+  "The all-purpose structured EUROCONTROL surveillance information exchange (ASTERIX)
+  is a set of documents defining the low level ('down to the bit') implementation
+  of a data format used for exchanging surveillance-related information and other ATM applications.
+  ASTERIX is designed for communication media with limited bandwidth. This is why it follows rules
+  that enable it to transmit all the information needed, with the smallest data load possible.
+
+  The ASTERIX library consists of several parts, each of them describing the encoding of information
+  related to a specific application.
```

Comment:
This message is copied verbatim from https://www.eurocontrol.int/asterix — are you sure that we're allowed to copy it?

Reply:
As I don't know, I've just sent an email to Eurocontrol on this, waiting for response.